File: app/src/main/java/com/chanda/lapaalap/splash/SplashActivity.java
package com.chanda.lapaalap.splash;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import androidx.appcompat.app.AppCompatActivity;

import com.chanda.lapaalap.R;
import com.chanda.lapaalap.fragments.FragmentMainActivity;


public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.splash_activity);
        goToNext();
    }

    private void goToNext() {
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent contentIntent = new Intent(SplashActivity.this, FragmentMainActivity.class);
                startActivity(contentIntent);
                finish();

            }
        },3000);
    }
}

File: app/src/main/java/com/chanda/lapaalap/SplashActivity.kt
package com.chanda.lapaalap

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.view.Window
import android.view.WindowManager
import android.view.WindowManager.LayoutParams.FLAG_FULLSCREEN
import androidx.appcompat.app.AppCompatActivity

@Suppress("DEPRECATION")
class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        window.setFlags(FLAG_FULLSCREEN, FLAG_FULLSCREEN)
        setContentView(R.layout.activity_splash)

        fun goToNext() {
                Handler().postDelayed({
                val contentIntent = Intent(this, MainActivity::class.java).setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
                startActivity(contentIntent)
                finish()
            }, 4000)
        }

        fun onCreate(savedInstanceState: Bundle?) {
            super.onCreate(savedInstanceState)
            setContentView(R.layout.activity_splash)
            goToNext()
        }


    }
}